### DISCUSSION TITLE
border around Swiss cross logo

### DISCUSSION THREAD

**Discussion by StephanPZ** [manuelbl/SwissQRBill.NET#51] [Q&A]
Hi,

I noticed that the white border around the Swiss cross logo is thicker than in all the QR bills and samples I have seen. Is there a setting to adjust this? 

![swiss-cross-border](https://user-images.githubusercontent.com/98612192/171741687-35fb6429-f246-42d4-b7bc-6a136007cd41.png)

Many thanks
Stephan

**Comment by manuelbl** [comment]
The library removes entire QR modules (sort of the pixels of the QR code) to make space for the Swiss cross. Thus, the white border width is affected by something similar to a rounding effect. It will differ, can be wider or smaller, all depending on the QR code version (the number of pixels used).

There is not configuration to change this behavior.

Is it a problem?

**Comment by StephanPZ** [comment]
Hello Manuel,
thanks for the explanation provided. 

Since the QR code is scannable, I think it is just a minor visual difference which does not disturb.

Regards
Stephan
